src/merkle.rs

```diff
@@ -2041,6 +2074,13 @@ fn test_get_merkle_tree_methods() {
     assert!(get_merkle_tree_len(1, 2).is_err());
     assert!(get_merkle_tree_len(4, 16).is_err());
     assert!(get_merkle_tree_len(1024, 11).is_err());
+    assert!(get_merkle_tree_len_generic::<U4, U0, U0>(16).is_ok());
+    assert!(get_merkle_tree_len_generic::<U1, U0, U0>(3).is_ok());
```

Comment:
You could also use `assert_eq!()` and test for the actual return value.

Reply:
Sure. Added mentioned tests. They will never be redundant